pkg/webui/components/file-input/index.js

```diff
@@ -101,8 +104,15 @@ export default class FileInput extends Component {
   handleChange(event) {
     const { maxSize } = this.props
     const { files } = event.target
+    const threeMb = 3 * 1024 * 1024
```

Comment:
```suggestion
    const warningSize = 10 * 1024 * 1024 // 10MB.
```

See here: https://github.com/TheThingsIndustries/lorawan-stack-support/issues/585#issuecomment-976298265